### PR TITLE
MET-110: Fix translations

### PR DESCRIPTION
**Pull request by SamirBoulil**
**Description (for Contributor and Core Developer)**

Fix translations mistakes made in: https://github.com/akeneo/pim-community-dev/pull/11746

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | Ok
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Akeneo/Tool/Bundle/MeasureBundle/Resources/translations/validators.en_US.yml [hunk 1]
```diff
@@ -9,20 +9,20 @@ pim_measurements:
                 is_required: 'The standard unit is required.'
                 should_be_in_the_list_of_units: 'The "%standard_unit_code%" standard unit code does not exist in the list of units for the "%measurement_family_code%" measurement family.'
                 cannot_be_changed: 'The standard unit code of the "%measurement_family_code%" measurement family cannot be changed'
-                operation_should_be_multiply_by_one: 'The standard unit code of the "%measurement_family_code%" measurement family should be a multiply by 1 operation'
+                operation_should_be_multiply_by_one: 'The standard unit code of the "%measurement_family_code%" measurement family should be a-multiply-by-1 operation'
```
Comment (tseho) [inline src/Akeneo/Tool/Bundle/MeasureBundle/Resources/translations/validators.en_US.yml]: are you sure about the first `-` ? look odd.

Comment (SamirBoulil) [reply src/Akeneo/Tool/Bundle/MeasureBundle/Resources/translations/validators.en_US.yml]: @camillebourjade 's recommendation

Comment (tseho) [reply src/Akeneo/Tool/Bundle/MeasureBundle/Resources/translations/validators.en_US.yml]: `a multiply-by-1` looks better to me, just wanted to be 100% sure it's not a typo.

Comment (tseho) [reply src/Akeneo/Tool/Bundle/MeasureBundle/Resources/translations/validators.en_US.yml]: (and I know it's camille recommendation, I saw her feedback in your previous PR)

Comment (camillebourjade) [reply src/Akeneo/Tool/Bundle/MeasureBundle/Resources/translations/validators.en_US.yml]: yes! sorry it's a typo! 🤭

**Pull request merged**